plugins/circleci/src/state/useBuilds.ts

```diff
@@ -68,8 +69,25 @@ export const transform = (
   });
 };
 
+export const useProjectSlugFromEntity = () => {
+  const { entity } = useEntity();
+  const [vcs, owner, repo] = (
+    entity.metadata.annotations?.[CIRCLECI_ANNOTATION] ?? ''
```

Comment:
I remember that the edge case when  ```entity.metadata.annotations?.[CIRCLECI_ANNOTATION]``` is missing wasn't handled well. Is it ok now?

Reply:
Yep, we have a safeguard in Router, if there is no annotation then the whole Route is gonna be inactive